host/sgx/region.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _REGION_H
+#define _REGION_H
+
+#include <openenclave/internal/debugrt/host.h>
+#include <openenclave/bits/sgx/region.h>
+#include <limits.h>
+
+typedef struct _oe_sgx_load_context oe_sgx_load_context_t;
+
+struct oe_region_context
+{
+    /* The current virtual address */
+    uint64_t vaddr;
+
+    /* Region table */
+    oe_region_t regions[OE_MAX_REGIONS];
+    size_t num_regions;
```

Comment:
```suggestion
    size_t region_count;
```
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md says:

> DON'T use abbreviations unless they are already well-known terms known by users (e.g., "app", "info"), or are already required for use by developers (e.g., "min", "max", "args"). Examples of bad use would be num_widgets instead of widget_count, and opt_widgets instead of option_widgets or optional_widgets.